src/core/src/sketch/minhash.rs

```diff
@@ -792,6 +792,43 @@ impl KmerMinHash {
         Ok(new_mh)
     }
 
+    pub fn downsample_scaled(&self, new_scaled: u64) -> Result<KmerMinHash, Error> {
```

Comment:
note: this might be something we want to spread throughout the code base after this. maybe create an issue?